.github/workflows/build.yaml

```diff
@@ -28,6 +28,11 @@ jobs:
     uses: ./.github/workflows/build-kanto-auto-deployer.yaml
     secrets: inherit
 
+  build-kantui:
```

Comment:
Change job name to build-blueprint-selector for example

Reply:
Ok e